### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by jpark952** [TheHive-Project/TheHive#1121]
Although not an essential feature, having the ability to configure your apps in Dark mode is widely sought after.

is there scope to add this feature to TheHive

**Comment by presianbg** [comment]
Hi,
You can use DarkReader browser extension. Result is OK:
![image](https://user-images.githubusercontent.com/2163486/65306439-04919700-db8e-11e9-8695-a5935a95eb13.png)

**Comment by milesflo** [comment]
I agree with this idea! Material made a great writeup on dark mode color theory https://material.io/design/color/dark-theme.html#properties

That would look great on this app since it should be used day/night and stay up on a monitor for long periods of time. Reduces eye strain and potential screen burn.

**Comment by nickbabkin** [comment]
+ to dark mode addition. Spending hours per day on TheHive and would like something that reduces eye strain. Browser plugins are not the option cause I'm kinda paranoid about them.
